tur/comcom64/build.sh

```diff
@@ -0,0 +1,28 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/dosemu2/comcom64
+TERMUX_PKG_DESCRIPTION="64bit command.com"
+TERMUX_PKG_LICENSE="GPL-3.0"
+TERMUX_PKG_MAINTAINER="@stsp"
+TERMUX_PKG_VERSION="0.3"
+TERMUX_PKG_SRCURL=git+https://github.com/dosemu2/comcom64
+TERMUX_PKG_GIT_BRANCH=master
```

Comment:
`master` it not valid. If it uses thunk version, you should checkout specific commit in `termux_step_post_get_source`. See the example below.

https://github.com/termux-user-repository/tur/blob/7e0f324ef3a38551182b9e68c134753586064a31/tur/vkmark/build.sh#L15-L36.

Reply:
Do I really need to do this, or maybe
just putting commit sha as branch is
enough?

Reply:
Also I see packages specify
`TERMUX_PKG_SHA256` in case of
the particular commit. What kind
of sha is this? I only know how to
calc sha of a release zip file.